packages/web-components/src/radio/radio.spec.ts

```diff
@@ -0,0 +1,290 @@
+import { expect, test } from "@playwright/test";
+import type { Locator, Page } from "@playwright/test";
+import { fixtureURL } from "../helpers.tests.js";
+import type { Radio } from "./radio.js";
+
+test.describe("Radio", () => {
```

Comment:
Formatting: single quotes. Have you run prettier on this?
I see `*.spec.ts` excluded in .prettierignore:
https://github.com/microsoft/fluentui/blob/5745de0ab6ab860f905c33d4430b857b8a72f27f/.prettierignore#L38-L39

Do you recall the reason? Can we re-enable it? Separate PR would be fine.

Reply:
Yeah, I'll enable it - I think that's outdated. I think perhaps one reason was due to the fact it was messing with spacing in output for HTML so we'll want to keep an eye on it.

Reply:
Updated!